### ISSUE TITLE
Extension tracking issue: bib: extension

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#431]
This issue serves to track Schema.org developments around the evolving bib: extension proposal.

See #699 for final review tracking. In particular #700 regarding "Agent"

At this point we do not have a well-polished proposal for reviewing hosted schema.org extensions, i.e. those that will show us as "bib.schema.org" or similar. 
- #429 tracks implementation of Extensions mechanism itself (i.e. python code and HTML/CSS UI)
- https://lists.w3.org/Archives/Public/public-schemabibex/2015Apr/0000.html "An initial proposal for bib.schema.org" within BibExtend CG

Issues to discuss (we can break out sub-issues as needed)
- [ ] is Schema.org CG (and steering group) happy with the term names requested for the extension?
- [ ] is there a high level mission statement indicating broad coverage of the extension?
- [ ] does it propose terms that are candidates for direct inclusion for the schema.org core?
- [ ] have we made the most of the opportunity to collaborate with BibFrame? see #642
- [ ] ... other questions?

Draft: https://www.w3.org/community/schemabibex/wiki/Bib.schema.org-1.0
# Summary of request terms (including revisions, issues)

(to be completed)
- [ ] Agent - Guha concerned about generality. #700
- [x] Chapter, now called BookChapter
- [ ] ...todo

**Comment by thadguidry** [comment]
It would be nice to simply use GitHub (and Git clients) Search to easily find same or similar named properties, types, etc. across core and extensions (to help with aligning extensions and core, etc).

Q: I think putting extensions themselves in a separate parent folder in the repository (instead of under /data ) might help even more with filtering mechanisms (path) in GitHub extensions and Git clients that many of us use ?

**Comment by danbri** [comment]
See https://lists.w3.org/Archives/Public/public-schemabibex/2015Apr/0057.html "Finalising bib.schema.org 1.0 proposal" thread.

**Comment by tmarshbing** [comment]
In quickly reviewing the proposal, several terms seem like ones that we would reject if we were reviewing this for inclusion in the core and that could be used by other extensions to mean different things than in this extension. E.g., Chapter can also mean http://en.wikipedia.org/wiki/Chapter_(religion) or similar, inSupportOf can be in support of things other than theses, and so on.

There are also some things that we would want to work through the structural relationships on. For example, how does the proposed Collection relate to the existing collection? How about artist vs. the existing byArtist? Even if we rename "Collection" and "artist" in the extension, these relationship questions would remain.

Given that this is the first extension we're reviewing, I'm not sure what sorts of issues we should and shouldn't be flagging. How do others think about the types of issues above?

**Comment by rvguha** [comment]
I had similar reactions. As the first extension, we should be extra
diligent.

guha

**Comment by rvguha** [comment]
As Tom pointed out, several of the term names (and concepts) are ambiguous
and/or replicate things in the core. Further, some of the terms proposed by
them should be in the core.

guha

**Comment by Dataliberate** [comment]
Hi

This first extension proposal is an interesting exercise both from the point of view of the proposers, with their natural domain biased views, and those in the core.

You are right to advise diligence, how this is approached will naturally establish patterns of practice for the future.

If you are watching the SchemaBibEx list<https://lists.w3.org/Archives/Public/public-schemabibex/2015Apr/0082.html>, you will see that we have reduced the proposals down to a hopefully not too controversial subset and are on the cusp of proposing it.  Having said that, some of the proposals we have deferred for later discussion (Containers<https://www.w3.org/community/schemabibex/wiki/Bib.schema.org-1.1#Containers> for example) may well be still of interest as generic core proposals at this time.

I am not unsurprised at the ‘it should be in the core’, ‘we have a potential name conflict’ comments.  How we work through these will need care.

Chapter for example makes total sense in the bib world, but has a totally different meaning to a Hells Angel.   Renaming to BookChapter may well be generally obvious but too constraining for bib uses, and CreativeWorkChapter too cumbersome.  Collection a candidate for the core?  But also used as an UpdateAction property name.

As to the breadth of the proposal I have been working under the following assumptions, taken from our email trail,  as to what could potentially be in our bib proposal:
Scope of an extension domain / splitting into several extensions
There are three aspects to this.

     *   Does a proposed term sit well in an extension with a particular name
     *   Is there a community ready, and most importantly willing, to focus on and maintain a particular subdomain
     *   Does the term in question satisfy 'a need to describe’ in the sponsoring [of an extension subdomain] community.

I find myself wearing two hats here, both chairing the group proposing an extension, and having broad interest in Schema and its development of a core general purpose vocabulary for describing stuff on the web.  How the core and extension proposing groups communicate and cooperate could impact greatly on the future success and utility of the vocabulary.


~Richard

On 25 Apr 2015, at 13:18, rvguha <notifications@github.com<mailto:notifications@github.com>> wrote:

As Tom pointed out, several of the term names (and concepts) are ambiguous
and/or replicate things in the core. Further, some of the terms proposed by
them should be in the core.

guha

**Comment by danbri** [comment]
@Dataliberate - can you talk us through the expected usage scenarios for 'Agent'? Historically we have resisted adding high-level supertypes over Person and Organization, preferring to repeat domain/range information on agent-related properties manually. 

There are scenarios such as sites publishing info where their records might not know whether e.g. Person vs Organization is most applicable. This can be seen e.g. at view-source: https://www.youtube.com/watch?v=jIFiubSPC2Q 

        <span itemprop="author" itemscope itemtype="http://schema.org/Person">
          <link itemprop="url" href="http://www.youtube.com/user/LeMundaneum">
        </span>

For this particular situation, an 'Account' type or similar could address the need. Do you have some other examples of how you'd expect 'Agent' to be used?

**Comment by danbri** [comment]
See also http://en.wikipedia.org/wiki/Intertwingularity "EVERYTHING IS DEEPLY INTERTWINGLED. In an important sense there are no "subjects" at all; there is only all knowledge, since the cross-connections among the myriad topics of this world simply cannot be divided up neatly." (Ted Nelson, 1974)

**Comment by danbri** [comment]
@Dataliberate @dbs where are we with the rdfs/rdfa config for this? I've been testing with a fake version derrived from bibliograph, but would like to update to latest while working out final python fixes. I don't see a link in https://www.w3.org/community/schemabibex/wiki/Bib.schema.org-1.0#Release_1.0_-_The_initial_release ...

**Comment by dbs** [comment]
@danbri My comics work is at https://github.com/dbs/schemaorg/tree/bibex_comics_coverart.

Note that the examples are in data/ext/bib/comics-examples.txt rather than in the data/ subdirectory because I assume that you're going to want to have extensions segregating their examples just as they do their RDFa, rather than cluttering up the core directory. But I guess I should open an issue to track that...

**Comment by danbri** [comment]
Looking at http://www.w3.org/community/schemabibex/wiki/Bib.schema.org -> http://www.w3.org/community/schemabibex/wiki/Bib.schema.org-1.0 and https://github.com/schemaorg/schemaorg/pull/461/files (which includes comics now)

* are any of the more general terms proposed for inclusion in the core? e.g. Agent, we assumed was a proposal to go into Core.
* I was just adding some mappings to MusicBrainz in and found http://schema.org/MusicReleaseFormatType - do these (eg. CassetteFormat) relate usefully to anything you have? (Or just more evidence that all topics are tangled with each other?)
* Do you have a running site with the latest proposals browsable?

**Comment by Dataliberate** [comment]
Looking at http://www.w3.org/community/schemabibex/wiki/Bib.schema.org -> http://www.w3.org/community/schemabibex/wiki/Bib.schema.org-1.0 and https://github.com/schemaorg/schemaorg/pull/461/files (which includes comics now)

  *   are any of the more general terms proposed for inclusion in the core? e.g. Agent, we assumed was a proposal to go into Core.

Not specifically, as this is the first stab at the extension/core proposal process.  However there are some candidates for such consideration:

  *   Agent as you point out.  It potentially could become a super type for additional Types beyond Organization & Person - BroadcastService and SoftwareApplication have previously been suggested.
  *   Collection —  Has many applications beyond the bib domain. Anywhere where an agent has created a collection of things virtual or physically.


  *   I was just adding some mappings to MusicBrainz in and found http://schema.org/MusicReleaseFormatType - do these (eg. CassetteFormat) relate usefully to anything you have? (Or just more evidence that all topics are tangled with each other?)

I didn’t know they were there, hidden in the music stuff.  Thinks like AudioBook (when we final decide how to describe one), Movie, etc. could also take advantage of these format types but the MusicReleaseFormatType name is a bit misleading.

You are right there is some intertwingling going on that needs clearing up around this area.  Just in books we have type (text, audio, e), format  (hardcover, paperback, fabric), physical (book, cassette, dvd, mp3, etc).  Is saying a book is large-print similar to saying it is paperback?

  *   Do you have a running site with the latest proposals browsable?

Our latest version is here, based upon sdo-scripts: http://sdo-bib.appspot.com

~Richard.

**Comment by danbri** [comment]
Thanks, Richard! FWIW the extensions-related code (and everything actually) from sdo-scripts is now merged into sdo-gozer. I've been working on closing out other issues lately so there are still issues with the extension code but it should be an improvement on what you see in sdo-scripts. In particular the homepage is now driven by JINJA2 (django-like) template files in templates/ including some basic treatment for extensions and test-build sites.

**Comment by ostephens** [comment]
Not sure where some of this discussion should happen (here, individual issues, sdc mailing list, bib extend mailing list?)

Given that 'Chapter' is currently defined by the extension as "One of the sections into which a book is divided" then 'BookChapter' would seem an appropriate term in place of the more ambiguous 'Chapter'

**Comment by mfhepp** [comment]
+1 to adding Agent to the schema.org core. There are cases where the publisher of data does not know whether an entity is a person or an organization. That was the motivation for defining gr:BusinessEntity (aka Agent) in GoodRelations.

**Comment by Dataliberate** [comment]
Picking up on Agent…

In support of the generic understanding in the bibliographic/library world that you need the ability to describe Agents — because often you can not ascertain with confidence that a creator, contributor, etc. is a Person or an Organisation, or [potentially] a service or a software application — we needed to come up with helpful information.

To that end Jeff Young has been doing some digging into the bibliographic data we have in OCLC WorldCat, and some of the industry processes and procedures.

A particular problem arrises when cross walking data from formats that are not specific about Person/Organisation or building directly on top of those formats.  Specifically this note published by the Library of Congress<http://www.loc.gov/marc/dccross.html> on taking Dublin Core format data into library (Marc) formats, is enlightening:
“Note: there is no way to specify whether the Contributor is a person or organization because it is not in the Dublin Core data. If it can reasonably be determined that the contributor is a person or organization, fields 700 1#$a (Added Entry--Personal Name) or 710 2#$a (Added Entry--Corporate Name) may be used.”

For this and other reasons the significant body of data that is in WorldCat and underpins the library community contains many examples of Agents.

To quantify this Jeff has analysed 337,136,762 bibliographic records in WorldCat and identified 71,794,268 Agents defined.  We would hope that future processing may reduce this 71 million number somewhat, as it is always preferable to identify Persons separate from Organizations, but the expectation is that a large proportion (10’s of millions) will remain undecided and therefore only definable as Agents.

Although specific to the library world I hope this information is helpful in understanding the expressed need for Agent as a super-type for Person & Organization and possibly others.

I would support, with others, Agent being adopted in the core.

~Richard

On 26 Apr 2015, at 22:25, Dan Brickley <notifications@github.com<mailto:notifications@github.com>> wrote:


@Dataliberate<https://github.com/Dataliberate> - can you talk us through the expected usage scenarios for 'Agent'? Historically we have resisted adding high-level supertypes over Person and Organization, preferring to repeat domain/range information on agent-related properties manually.

There are scenarios such as sites publishing info where their records might not know whether e.g. Person vs Organization is most applicable. This can be seen e.g. at view-source: https://www.youtube.com/watch?v=jIFiubSPC2Q

    <span itemprop="author" itemscope itemtype="http://schema.org/Person">
      <link itemprop="url" href="http://www.youtube.com/user/LeMundaneum">
    </span>


For this particular situation, an 'Account' type or similar could address the need. Do you have some other examples of how you'd expect 'Agent' to be used?

**Comment by danbri** [comment]
I have merged #461 into sdo-gozer. I don't expect this to end the conversation underway here, but it is important for integration testing and simplicity to be working towards a single sanely-functioning release candidate branch. Given the changes queued up, there are both vocabulary and navigation/software considerations. Although these are all converging on the sdo-gozer branch, let's try to keep clear in our discussion when we're discussing site navigation / structural features vs schema/vocab issues.

FWIW here are the types and properties in the files I've just merged. I'll delete the old bibdemo.rdf as obsoleted by this real proposal.

$ rdfa bsdo-1.0.rdfa |grep "#Class"
    <http://schema.org/Organization> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2000/01/rdf-schema#Class> .
    <http://schema.org/Chapter> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2000/01/rdf-schema#Class> .
    <http://schema.org/Atlas> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2000/01/rdf-schema#Class> .
    <http://schema.org/Person> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2000/01/rdf-schema#Class> .
    <http://schema.org/Thesis> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2000/01/rdf-schema#Class> .
    <http://schema.org/Newspaper> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2000/01/rdf-schema#Class> .
    <http://schema.org/Agent> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2000/01/rdf-schema#Class> .

$ rdfa bsdo-1.0.rdfa |grep "#Property"
    <http://schema.org/creator> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/1999/02/22-rdf-syntax-ns#Property> .
    <http://schema.org/translationOfWork> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/1999/02/22-rdf-syntax-ns#Property> .
    <http://schema.org/author> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/1999/02/22-rdf-syntax-ns#Property> .
    <http://schema.org/pageStart> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/1999/02/22-rdf-syntax-ns#Property> .
    <http://schema.org/pagination> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/1999/02/22-rdf-syntax-ns#Property> .
    <http://schema.org/workTranslation> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/1999/02/22-rdf-syntax-ns#Property> .
    <http://schema.org/copyrightHolder> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/1999/02/22-rdf-syntax-ns#Property> .
    <http://schema.org/contributor> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/1999/02/22-rdf-syntax-ns#Property> .
    <http://schema.org/translator> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/1999/02/22-rdf-syntax-ns#Property> .
    <http://schema.org/inSupportOf> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/1999/02/22-rdf-syntax-ns#Property> .
    <http://schema.org/publishedBy> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/1999/02/22-rdf-syntax-ns#Property> .
    <http://schema.org/pageEnd> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/1999/02/22-rdf-syntax-ns#Property> .

**Comment by danbri** [comment]
This won't go out as part of sdo-gozer, we'll do separate updates once the first wave of extensions are ready.

**Comment by danbri** [comment]
List of type names per current candidate schema: https://github.com/schemaorg/schemaorg/pull/461/files#diff-c50396bf2a61d4cf44f88edfcd4c436dR29

**Comment by ppKrauss** [comment]
Hi, as the classic [FRBR reference model](https://en.wikipedia.org/wiki/Functional_Requirements_for_Bibliographic_Records) shows the need of *Expression* concept, we can add it to unify and generalize translation and versioning ... something like

```
workExpression > workTranslation
workExpression > workVersion > softwareVersion
```
PS: remembering that the FRBR's *Work* concept is the SchemaOrg's [CreativeWork](http://schema.org/CreativeWork) concept. <br/>The *bib*'s [workTranslation](http://bib.schema.org/workTranslation) **is not** "a kind of CreativeWork" like [Article](http://schema.org/Article), **but** "a kind of FRBR's *Expression*".

![](https://upload.wikimedia.org/wikipedia/commons/thumb/8/80/FRBR-Group-1-entities-and-basic-relations.svg/270px-FRBR-Group-1-entities-and-basic-relations.svg.png)

**Comment by RichardWallis** [comment]
Hi @ppKrauss 

Thanks for chiming in on this long dormant thread.

The '_do we implement FRBR in Schema.org_' question was discussed in depth and in detail in the [Schema Bib Extend Community group](https://www.w3.org/community/schemabibex/), prior to the proposal and introduction of the [bib.schema.org](http://bib.schema.org/) extension in 2015.

In [very] brief summary, it was concluded that the very structured and controlled relationships between concepts, and the way it has been implemented in the bibliographic, especially library, world, was not suitable for the generic broad set of use cases that Schema.org addresses.

If you check [the example model](https://www.w3.org/community/schemabibex/wiki/Examples/mylib) used to help the development of the bib extension, you will see that all elements in the FRBR model are considered to representable as Schema.org CreativeWorks "_The most generic kind of creative work, including books, movies, photographs, software programs, etc._".  

The differentiation between, for example what would be a FRBR Work and a FRBR Manifestation, are handled by using the Schema.org construct of multiple-types.  ie. A Manifestation is represented by defining an entity as _both_ a **CreativeWork** and a **Product**.

Also, from a practical and pragmatic point of view, the concepts of FRBR are well outside of the needs of most of the practitioners sharing information to aid discovery on the web - one of the main rationales behind the introduction of Schema.org

Relating this to personal experience in helping libraries share Schema.org data about their resources, it works quite well in practice.  If it is need to relate resources using FRBR concepts and terms, this is achieved by using bibliographic specific vocabularies (MARC, RDA, BIBFRAME) internally within the library domain whilst sharing information in a more generic, broadly consumable (Schema.org) format via their web presence.

I see no broad appetite in the community for a change in this.

~Richard.

**Comment by ppKrauss** [comment]
Hi, thanks @RichardWallis , good explanation and historic review. 

Perhaps we need to move a little the discussion out from FRBR, and, for a another consensus check, focus in the *common ancestor* problem/solution.

------

About your final summary, *"no broad appetite in the community for a change in this"*, 

* is not "to change" but "to extend": there are, in use at SchemaOrg, `workExpression`  and `softwareVersion` (see [illustration above](#issuecomment-390649130)), so <br/>the proposed "extension" is a [**common ancestor**](https://en.wikipedia.org/wiki/Lowest_common_ancestor), `workExpression`. 

* at 2015's examples there are only one Work, [A6](https://www.w3.org/community/schemabibex/wiki/Examples/mylib/A6) and one Expression, [A0](https://www.w3.org/community/schemabibex/wiki/Examples/mylib/A0), as "abstract objects". All other examples are "concrete objects" (FRBR's Itens and Manifestations). <br/>No comparison (two abstract objects of the same class) is possible to discuss here.

* before to check the *broad appetite*, we need first offer an interesting thing (with better taste!) to the community.  ;-)<br/> Imagining that we need a better set of SchemaOrg classes to express the semantic of these "abstract objects".

----

About the the differentiation between "what would be a FRBR Work and a FRBR Manifestation", we can use the same *common ancestor* argument: [MediaObject](http://schema.org/MediaObject) is used in SchemaOrg as a variant of the FRBR's Manifestation concept. 

... But remember that FRBR's Manifestation is a "concrete object",  and FRBR's Work and Expression are "abstract".  My focos is only in abstract ones, and I am imagining that there are broad appetite for "abstract objects" and its metadata, particularly *Expression*-like  ones. 

The first step for any community to use metadata of an object, is to identify the object...  There are a broad appetite for document-identifiers based on metadata (transparent IDs), it is a good use case... Examples:

* transparent IDs: [ELI](https://en.wikipedia.org/wiki/European_Legislation_Identifier), [ECLI](https://en.wikipedia.org/wiki/European_Case_Law_Identifier) and [URN LEX](https://en.wikipedia.org/wiki/Lex_(URN)#Transparent_identifiers) use the *abstract view of document*. As a normative principle, it requires (to be a transparent ID) that the metadata used for identificator construction is mapped from original content &mdash; so is mapped from eg. microdata marked title, published date, abstract, etc.  

*  centralized opaque IDs, like in [DOI system](https://en.wikipedia.org/wiki/Digital_object_identifier): DOI use the [Indecs Content Model](https://en.wikipedia.org/wiki/Indecs_Content_Model), so it use the "abstract view" when identifing documents. No matter about *Manifestation* (eg. XML and PDF formats of [this article](doi.org/10.1371/journal.pone.0023630) have the same DOI).

* full-decentralized... [DID](https://w3c-ccg.github.io/did-spec/), there are a broad community and interest in decentralization, so "transparent ID" (like ELI or URN LEX) in a blockchain (as [Blockstack's BNS](https://blockstack.org/whitepaper.pdf)).  "Transparent ID" is a naming schema based in semantic metadata... And the broadly consumable semantic metadata are born here, at SchemaOrg.

PS: ISBN and [ISSN](https://pt.wikipedia.org/wiki/International_Standard_Serial_Number#Media_vs_content) are based on the "concrete view" (eg. books as old paper/CD/microfilm products not as FRBR's works). Only after ~30 years sleeping the ISSN created ISSN-L for the abstract view identification.

**Comment by RichardWallis** [comment]
By change, I intended to also imply extension in this area.

Looking at implementing or extending features in Schema.org tends to be a pragmatic, use-case driven process.  This is what led to the current [Schema.org] model where the generally useful properties of CreativeWork (_exampleOfWork_, _workExample_, _translationOfWork_, _workTranslation_, _hasPart_, _isPartOf_) can be applied to describe a broad range of relationships, including those in the constrained four-layer FRBR model.

For example there are no limits to the number of exampleOfWork/workExample relationships in a graph of related entities - the bibliographic community, which I have spent a happy quarter of a century being associated with, are yet in consensus about what to do (in modelling terms) about the so-called super-work (the Work that the movie (Work) and the book (Work) are both an example of).

Taking a pragmatic approach, from your suggestion, leads me to the following thought process:

- Using the established patterns within Schema.org, whatever you suggest to represent a FRBR Expression equivalent would most probably have to be a subtype of Creative Work.
Be it totally abstract (Tolstoy's story) or very concrete (an individual copy of the book War and Peace on a bookshop shelf), in Schema.org it is described using a CreativeWork type or subtype.  For the physical properties, associated with what you describe as concrete, other Types are added such as Product. 
- If we were to propose a schema.org type that approximates to a FRBR Expression, what new properties would be required to supplement those inherited from CreativeWork, and what purpose would they serve towards aiding visibility and discoverability on the web.
- If we implemented a schema.org type that approximates to a FRBR Expression, which websites would use it, which services and sites would actively consume it, and what benefits would they hope to gain from doing so.

The main reasons I say that I see no appetite for change (or extension) in this area, is that the relationships between, and properties of, the entities within your area of concern can be described (sufficient to aid visibility and discovery on the web) using the vocabulary as it is currently constituted.  
Utilising CreativeWork (or a subtype) on its own for describing what approximates to a Work/Expression, and in combination with other type(s) to describe what approximates to a Manifestation/Item is sufficiently detailed enough to satisfy the needs of the vast majority of Schema.org implementers to whom the subtle differences in he concepts within the FRBR model are unknown and not relevant.

Please do misunderstand me, I see the need within the metadata for bibliographic, and other creative work, communities for extracting and recording detailed data about resources, abstract and/or concrete, utilising ontologies and vocabularies constructed to satisfy the needs expressed by the FRBR model.    What I do not see is the broad benefit, (in visibility, discovery results, or ease of adoption) of adding such domain specific detail to the generic cross domain vocabulary that is Schema.org.

**Comment by philbarker** [comment]
+1 for what @RichardWallis is saying: schema.org doesn't need to replicate FRBR; people who need  FRBR can use FRBR.

(Also, I'm happy to see that I'm not the only person who occasionally misses the word 'not' from a sentence: 'Please do misunderstand me..')

**Comment by RichardWallis** [comment]
Thanks Phil for spotting my [all to common] typo - 'not' now inserted into the text ;-)

**Comment by ppKrauss** [comment]
Hi @RichardWallis and @philbarker , thanks again the explanation and directions. Sorry to not noticed that  `exampleOfWork`/`workExample` aren't illustrations, but serious properties, and that can be used with "abstract objects"... Is not so direct to model all with it (insted to easy subtyping), I will try, will test with my use cases, and back here later with my findings to check the correctness of my homework.
